selfdrive/ui/translations/main_ja.ts

```diff
@@ -309,7 +309,7 @@
     <name>MapETA</name>
     <message>
         <source>eta</source>
-        <translation>予定到着時間</translation>
+        <translation>到着予定時間</translation>
```

Comment:
This might be too long for nav, but let's fix in another PR. this isn't a regression